swengmgmt/commands.py

```diff
@@ -134,7 +134,8 @@ def _printItemized(self, student_list):
             print util.bold(str(student))
             print "  Team: %s" % util.red_if_none(student.team_name)
             print "  Github ID: %s" % util.red_if_none(student.github_id)
-            print "  Team Clone URL: %s" % util.red_if_none(student.team.repo_ssh_url)
+            if student.team:
```

Comment:
Good catch! I remember running into this issue last year, but forgot to commit back the changes.